modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -20,6 +20,7 @@ class ApplicationController < ::ApplicationController
       skip_after_action :set_csrf_header
       before_action :authenticate, except: %i[schema] # rubocop:disable Rails/LexicallyScopedActionFilter
       before_action :validate_json_format, if: -> { request.post? }
+      before_action :validate_header_values_format, if: -> { header_request? }
```

Comment:
Adding this call inside the `validate_veteran_identifers` method produced some weird Rubocop issues with the overall complexity of that method.  Which I do think makes sense since it is already doing a lot.  Since I wanted this to run first thing I feel it made just as much, if not more, sense to set it here right before that method runs since we want the headers to have valid values before going any further in the chain.  But open to suggestions if this looks off, or ends up being roblematic in a way I did not catch.